src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -113,6 +113,11 @@ class ShenandoahRegionIterator : public StackObj {
 class ShenandoahHeapRegionClosure : public StackObj {
 public:
   virtual void heap_region_do(ShenandoahHeapRegion* r) = 0;
+  // Default to global value ShenandoahParallelRegionStride, When it is set to 0,
+  // ShenandoahHeap::parallel_heap_region_iterate will derive a reasonable value based
+  // on active worker threads and number of regions.
+  // For some lumpy workload, the value can be overridden for better task distribution.
```

Comment:
Again, excessive. You can just drop the comment; its purpose is obvious from the code.